### PR TITLE
Release to Staging

### PR DESCRIPTION
**Pull request by trashutz**
Changes:
- Code of conduct as file instead of link #300
- Changes for event pages #302 
- Fixes for English pages #301

### CODE REVIEW

File: config/_default/menus.en.toml [hunk 1]
```diff
@@ -17,72 +17,72 @@ url = 'en/events/'
...
+#weight = 32
+#
+#[[main]]
+#name = "Mattstedt"
+#url = "en/mattstedt/"
+#parent = "events"
+#weight = 33
+#
+#[[main]]
+#name = "Open Doors Day"
```
Comment (hnolzen) [inline config/_default/menus.en.toml:38]: I guess it is just "Open Day" or "Open House".

**Review by hnolzen — COMMENTED** [review]
Looks good to me! 

I am just not sure about the translation of "Tag der offenen Tür".

File: config/_default/menus.en.toml (hunk 1 above)
Comment (trashutz) [reply config/_default/menus.en.toml:38]: Yeah, it was a bit confusing for me too...

https://en.wikipedia.org/wiki/Doors_Open_Days
- "Doors Open Days"
- "Open House" (seems to be for the US)
- "Open Days"
- but also "Open Doors events in Wales"

I have a british coworker.. Maybe I can ask him.

Comment (hnolzen) [reply config/_default/menus.en.toml:38]: I was not aware that Open Doors Day is also possible. Maybe let's resolve it for now. It is a very minor issue. We can always rename it later if necessary.

Comment (trashutz) [reply config/_default/menus.en.toml:38]: Ok. Can you finish the review, so we can merge?

**Review by hnolzen — APPROVED** [review]

**Pull request merged**